dojo/settings/settings.dist.py

```diff
@@ -1283,6 +1283,7 @@ def saml2_attrib_map_format(dict):
     "Kiuwan SCA Scan": ["description", "severity", "component_name", "component_version", "cwe"],
     "Rapplex Scan": ["title", "endpoints", "severity"],
     "AppCheck Web Application Scanner": ["title", "severity"],
+    "AWS Inspector2 Scan": ["title", "severity"],
```

Comment:
Hi @Maffooch ,
I am not sure if title and severity is sufficient here as Inspector could scan multiple (different) container with the same CVE. Thus, I would recommend to also use the description or information provided in "resources". (e.g. "imageHash": "sha256:c667ad018b538adadb5bd1db75836c8561b23bc7d31d9cd8941fa74f5df93802",)

Reply:
I can do `title, severity, description`. In most cases, the `unique_id_from_tool` will be used instead